public/js/utils.js

```diff
@@ -126,17 +126,25 @@ var prepareReport = function (data){
   var fails = [];
   var succs = [];
   // remove the site folder
-  var data = data.slice(1, data.length)
+  var data = data.slice(1, data.length);
+
   $.each(data, function(i, item){
-    if(item.status.indexOf('success') === -1 && item.status.indexOf('created') === -1 && item.status !=='') {
-      item.status_code="fail";
-      fails.push('fail');
+    if(item.status){
+      if(item.status.indexOf('success') === -1 && item.status.indexOf('created') === -1 && item.status !=='') {
+        item.status_code="fail";
+        fails.push('fail');
+      }
+      else {
+        item.status_code="success";
+        succs.push('suc');
+      }
     }
     else {
-      item.status_code="success";
+      item.status_code="unknown";
       succs.push('suc');
```

Comment:
Good question - because otherwise it would not show in the list. I will revisit this later - the data is not all that good and needs to change - till then this just makes it work